tests/d3d12.c

```diff
@@ -47625,6 +47646,115 @@ static void test_virtual_queries(void)
     destroy_test_context(&context);
 }
 
+static void test_vrs(void)
+{
+    D3D12_GRAPHICS_PIPELINE_STATE_DESC pso_desc;
+    ID3D12GraphicsCommandList5 *command_list;
+    bool additional_shading_rates_supported;
+    ID3D12PipelineState *pipeline_state = NULL;
+    struct test_context_desc desc;
+    struct test_context context;
+    ID3D12CommandQueue *queue;
+    unsigned int i;
+    HRESULT hr;
+#if 0
+    void main(in float4 vPos : SV_POSITION, out float4 o0 : SV_Target0)
+    {
+        o0 = float4(ddx(vPos.x) / 4.0, ddy(vPos.y) / 4.0, 0.0, 0.0);
```

Comment:
Dividing by 4.0 here relies on very specific rounding being used since 0.25 or 0.5 cannot be represented well. Some other tests have had this problem when I needed to fix up the rounding. I suggest dividing by 255.0 or something instead.

Reply:
Makes sense. Fixed.